dashboard/test/controllers/api/v1/pd/workshops_controller_test.rb

```diff
@@ -89,6 +89,30 @@ class Api::V1::Pd::WorkshopsControllerTest < ::ActionController::TestCase
     assert_equal workshop_2.id, response[0]['id']
   end
 
+  test 'exclude_summer excludes local summer workshops if included' do
+    sign_in @organizer
+
+    create :pd_workshop, :local_summer_workshop, organizer: @organizer
+
+    get :index, params: {exclude_summer: 1}
+    assert_response :success
+    response = JSON.parse(@response.body)
+    assert_equal 1, response.length
+    assert_equal @workshop.id, response[0]['id']
+  end
+
+  test 'includes local summer workshop if exclude_summer not present' do
```

Comment:
nit: name these 2 test cases consistently, e.g. `[includes|excludes] local summer workshop when exclude_summer is (not) present`